packages/react-query-devtools/package.json

```diff
@@ -36,5 +36,14 @@
     "@tanstack/match-sorter-utils": "^8.0.0-alpha.82",
     "@types/use-sync-external-store": "^0.0.3",
     "use-sync-external-store": "^1.2.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
+  },
+  "peerDependenciesMeta": {
+    "react-dom": {
+      "optional": true
+    }
```

Comment:
for the devtools, I don't think `react-dom` is optional, as it's a browser ui :)

Reply:
Noted!